ground/src/main/java/com/google/android/ground/ui/map/gms/mog/MogClient.kt

```diff
@@ -91,6 +91,7 @@ class MogClient(val collection: MogCollection) {
    * Returns a [Flow] which emits the tiles for the specified tile coordinates along with each
    * tile's respective coordinates.
    */
+  // TODO(#1596): Cache the jpeg files locally instead of downloading each time.
```

Comment:
This isn't necessary for online visualization since the GMS caches tiles itself.

It also isn't necessary here for offline use because the plan is to download the files to the file system as JPEG, and to use a simple file-based overlay instead of the MOG one to visualize.